### ISSUE TITLE
InsertFromFile

### ISSUE THREAD

**Issue by kevin-vanderwende** [NetOfficeFw/NetOffice#290]
I'm not sure if this is the place to ask this or not.  I have a project I've been working on for a while now.  I merge hymns into one presentation.  I have a slide that goes before each song where I add the number and title.  That slide has an orange border along the bottom.  The problem is when I do InsertFromFile that style is picked up by the next slide I insert even though it shouldn't.  I also merge in the sermon outline but that changes from week to week.  This past week there were some objects that weren't showing up.  I had been looping through the slides in the source file and applying them to the slides in the new file.  It works well for keeping the title slide and its formatting but I think that is where my issue with missing objects happened.

Up until this week I had been doing the interop myself but this package makes it so much easier to deal with.  I suspect the issue I am having is a product of how InsertFromFile works.  I had the same problem with doing the interop myself.  This package gets me a lot closer to where I need to be.

I've tried copy/paste as well with similar results.

This is what I had been doing with slide properties.  slide is my new slide and srcSlide is from the original presentation.

                slide.Design = srcSlide.Design;
                slide.ColorScheme = srcSlide.ColorScheme;
                slide.Background.Fill.ForeColor.RGB = srcSlide.Background.Fill.ForeColor.RGB;
                
                slide.SlideShowTransition.AdvanceOnClick = srcSlide.SlideShowTransition.AdvanceOnClick;
                slide.SlideShowTransition.AdvanceOnTime = srcSlide.SlideShowTransition.AdvanceOnTime;
                slide.SlideShowTransition.AdvanceTime = srcSlide.SlideShowTransition.AdvanceTime;
                slide.SlideShowTransition.Duration = srcSlide.SlideShowTransition.Duration;
                slide.SlideShowTransition.EntryEffect = srcSlide.SlideShowTransition.EntryEffect;
                slide.SlideShowTransition.Speed = srcSlide.SlideShowTransition.Speed;

**Comment by kevin-vanderwende** [comment]
To close the loop here is what I've found.  I am saving as PPT because I am
combining PPT and PPTX.  If I merge a few presentations together in a test
app and save as PPTX everything is very close.  If I save as PPT colors and
formatting are off.  I've tried saving the combined file as PPTX and I have
a series of other problems.

This was not something I had considered at first but the only difference
between my main app and test was document type.  In the presentation that
was giving me problems there is a graphic that is blue.  If I save as PPTX
it is blue.  If I save as PPT it changes to orange.
